packages/mgt-components/src/graph/types.ts

```diff
@@ -22,6 +22,22 @@ export type IDynamicPerson = (MicrosoftGraph.User | MicrosoftGraph.Person | Micr
    * @type {string}
    */
   personImage?: string;
+
+  /**
+   * personDetails.presenceActivity for text use
+   *
+   *
+   * @type {string}
+   */
+  presenceActivity?: string;
```

Comment:
I wonder if there is a way to handle this directly in the person component without adding the presence information on the this type - similarly how email is handled?

Reply:
Not sure if this is exactly what you meant, but I tried just adding it to just the personDetails section. The only problem is losing the presence type, but I could also just update the filter if you think its necessary. 
09df062